library/analytics/proguard-rules-adobe.pro

```diff
@@ -1 +1,5 @@
--dontwarn com.adobe.mobile.**
\ No newline at end of file
+-dontwarn com.adobe.mobile.**
+
+# Rules for SnowPlow
```

Comment:
split these out into a separate proguard file. it helps keep things organized as more libraries add varying rules.

Reply:
and you would just add a second `consumerProguardFiles` line under `defaultConfig`